signer/src/ws_server/mod.rs

```diff
@@ -43,17 +46,48 @@ impl From<ws::Error> for ServerError {
 	}
 }
 
+/// Builder for `WebSockets` server
+pub struct ServerBuilder {
+	handler: Arc<IoHandler>,
+}
+
+impl Default for ServerBuilder {
```

Comment:
think there's something to be said for making `default` the canonical means of naming/refering to parameter-less construction functions; it's got a stronger semantic notation given it's a trait rather than Just Another Consistently Named Function.